routes/api.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (app, db) ->
```

Comment:
With the change I suggested on the requireFu lines we should change this to be 

```coffee
module.exports = (app, api, db) ->
```

and remove the content from lines 2 and 4.